modules/space/rotation/spicerotation.cpp

```diff
@@ -52,6 +52,12 @@ namespace {
         "The time frame in which the spice kernels are valid."
     };
 
+    constexpr openspace::properties::Property::PropertyInfo FixedDateInfo = {
+        "FixedDate",
+        "Fixed Date",
+        "A time to lock the roation to."
```

Comment:
- Tiny spelling mistage:  roation -> rotation
 - I'd suggest to add a sentence describing that if he string is set to the empty string, it disables the fixed rotation